multicircuit_test.ipynb

```diff
@@ -0,0 +1,190 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit import QuantumCircuit, transpile\n",
+    "from qiskit_ionq import IonQProvider\n",
+    "from math import pi\n",
+    "from pprint import pprint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
```

Comment:
The outputs here have a lot of errors, are you just staging this for now, not asking for a review?

Reply:
oh im still developing on this ill move it to a draft pr